source/frontend/package.json

```diff
@@ -16,6 +16,7 @@
     "convert": "4.4.2",
     "dequal": "2.0.2",
     "esri-leaflet": "3.0.14",
+    "dms-conversion": "3.1.3",
```

Comment:
do we need this extra library? I believe we implemented the DML -> Lat/Long logic internally but perhaps there is value in replacing that with a library? I ask because I did not see the original DML logic removed in favor of this library